airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalAttemptExecution.java

```diff
@@ -139,7 +139,10 @@ public OUTPUT get() {
       }
 
       LOGGER.info("Executing worker wrapper. Airbyte version: {}", airbyteVersion);
-      saveWorkflowIdForCancellation(airbyteApiClient);
+      AirbyteApiClient.retryWithJitter(() -> {
+        saveWorkflowIdForCancellation(airbyteApiClient);
+        return null;
```

Comment:
Can we remove this return null?

Reply:
I looked into this and I think the alternatives are messier.

We can turn the interface into a Runnable, which doesn't expect an return value. However Runnables do not support checked exceptions, which means we would have to catch and recast the exception thrown by `saveWorkflowIdForCancellation` into a RTE.

Give that, I think it's simpler to keep this as is!